docs/logging-and-monitoring/health-metrics.md

```diff
@@ -0,0 +1,170 @@
+# Node's status and health
+
+This document focuses only on health-related Prometheus metrics: discusses their meaning in detail,
+and suggests some potential "Node status dashboard" ideas.
+
+## Health's components
+
+There is no single scalar value that would fully capture the concept of Node's "health". We can
+consider the following aspects:
+
+### Application's lifecycle status
+
+The Node, as an application, is not fully operational right after starting its process:
+- it has to initialize its sub-systems (memory strutures, worker threads),
+- if started in a pristine state, then it also has to initialize its persistent Ledger (a.k.a. run 
+  the Genesis Transactions predefined for the Network).
+
+This "spin-up" lifecycle phase - under normal circumstances - is shorter than typical Prometheus
+scraping interval, so it does not publish any dedicated metrics. It would also be cumbersome to even
+expose the Prometheus endpoint during this phase (due to technical details of our current spin-up
+implementation).
+
+This means:
+- if the Node's process has opened the Prometheus port and returns _any_ metric, then the spin-up
+  phase was finished successfully and the Node now actively tries to participate in its Network,
+- if the Node's process runs, but has not yet opened the Prometheus port, then it may have
+  encountered some specific problems (with the environment, or with its configuration, or with the
+  Genesis Transactions execution), which requires inspecting the Node's logs.
+  _Please note that such problems would be hard to unambiguously capture in the metrics anyway._
+
+### Ledger sync / commit status
+
+After the spin-up, one of the Node's primary goals is to keep its local Ledger "synced". Nodes
+constantly exchange their Ledger status information with one another and compare the following two
+metrics:
+
+rn_sync_current_state_version
+: The state version of the last transaction written to the local Ledger (either coming from a BFT
```

Comment:
_(note from author)_

This is a syntax for a "definition block" in markdown.
I now see that GH does not render it in any special way, but e.g. IntelliJ does.